### ISSUE TITLE
Dynamic type does not work on the site assembly screen

### ISSUE THREAD

**Issue by guarani** [wordpress-mobile/WordPress-iOS#22190]
### Expected behavior

The screen shown after creating a new site should handle larger text sizes. =

### Actual behavior

This screen clips many elements when text size is increased

### Steps to reproduce the behavior
1. Create a new site
2. Stop on the screen that says "Kudos, your site is live!"
3. Enlarge system or app-wide text sizes
4. Notice clipped text and images

##### Tested on iPhone 11, iOS 17.1.1, Jetpack iOS `trunk` a20e55f250a73d5d6a5fdc831973567a566a96cf

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
